pdsfile/pds4file/rules/cassini_uvis_solarocc_beckerjarmak2023.py

```
@@ -0,0 +1,272 @@
+
+##########################################################################################
+# Update the global dictionary of subclasses
+##########################################################################################
+
+pds4file.Pds4File.SUBCLASSES['cassini_uvis_solarocc_beckerjarmak2023'] = cassini_uvis_solarocc_beckerjarmak2023
+
+##########################################################################################
+# Unit tests
+##########################################################################################
+
+import pytest
```

Ruff (0.12.2)

Module level import not at top of file

(E402)